src/systems/ec/implementations/vanilla/packages/ec-component-gallery/ec-component-gallery.scss

```diff
@@ -106,6 +106,12 @@
     padding: $ecl-spacing-xs $ecl-spacing-s;
   }
 
+  .ecl-gallery__footer {
+    border-top: 1px solid $ecl-color-grey-10;
+    margin-top: calc(#{$ecl-spacing-xs} - 2px);
```

Comment:
we should not tweak the spacing values like this.
I guess that here the issue comes from the padding around the "view all" link, right? If there are inconsistencies like this, it should be reported to the designer (on the spec page or in the ticket)